drivers/config.go

```diff
@@ -45,6 +46,8 @@ func (c Config) MustInt(key string) int {
 		integer = t
 	case float64:
 		integer = int(t)
+	case string:
+		integer, _ = strconv.Atoi(t)
```

Comment:
Not keen on ignoring the errors here. If this error occurs then the key was set by a user that expects it to parse correctly and become an integer. If that doesn't work because there's a type/some other problem we should have the program fail I think? That's in essence the entire point of MustInt, it -HAS- to be an int or it should fail.

Reply:
Thanks, my bad. I've fixed this issue in 6722d57ab31fffadaf70e219e0deb7d4ac4406c8